src/components/Album.js

```diff
@@ -30,82 +35,332 @@ import AddToWishlistButton from './layout/AddToWishlistButton/AddToWishlistButto
+                  </Box>
+                </Box>
+
+                <Typography className="title" variant="h5">
+                  Review:
+                </Typography>
+                {reviewList.map((val) => (
+                  <div className="rate-card">
+                    {isEdit === val._id ? (
+                      <Box className="input-main">
+                        <TextField
+                          id="outlined-basic"
+                          label="Enter title"
+                          value={editObj.title}
+                          onChange={(e) =>
+                            setEditObj({ ...editObj, title: e.target.value })
+                          }
+                          variant="outlined"
+                        />
+                        <TextField
+                          id="outlined-multiline-static"
+                          label="Enter comment"
+                          multiline
+                          value={editObj.comment}
+                          onChange={(e) =>
+                            setEditObj({
+                              ...editObj,
+                              comment: e.target.value,
+                            })
+                          }
+                          rows={2}
+                        />
+                        <TextField
+                          id="outlined-basic"
+                          label="Enter rating between 1 to 5"
+                          value={editObj.rating}
+                          onChange={(e) =>
+                            setEditObj({ ...editObj, rating: e.target.value })
+                          }
+                          min={1}
+                          max={5}
+                          variant="outlined"
+                        />
+
+                        <Box sx={{ display: 'flex', gap: '10px' }}>
+                          <Button
+                            className="btn delete-btn"
+                            onClick={() => {
+                              setIsEdit('');
+                              setEditObj('');
+                            }}
+                          >
+                            Cancel
+                          </Button>
+                          <Button
+                            className="btn edit-btn"
+                            onClick={handleEditReview}
+                          >
+                            Save
+                          </Button>
+                        </Box>
+                      </Box>
+                    ) : (
+                      <>
+                        <Typography className="rate-data">
+                          {val.title}
+                        </Typography>
+                        <Typography>Rate: {val.rating}</Typography>
+                        <Typography>
+                          Posted on{' '}
+                          {new Date(val.createdAt).toLocaleDateString()}
```

Comment:
It would be nice to also include "Posted by: \<user name>". That may require an update on the backend to populate that data (you can feel free to do it or ask backend team for help getting that change in if needed)

Reply:
considering there is a demo on Monday, I will defer this for now and will take care of it in the future